capstone/src/main/resources/static/script.js

```diff
@@ -416,3 +403,20 @@ function closeForm() {
 	document.getElementById("organization").style.display = "none";
 }
 
+/* function to generate divs for the calendar */
+function createCalendar() {
+  const calendar = document.getElementById("calendar");
+  for (var i = 0; i < 14; i++) {
+    var nextDay = new Date();
+    var today = new Date();
+    nextDay.setDate(today.getDate() + i);
```

Comment:
date has some different formatting that might be helpful to cut down on the amount of info given with the time